northstar/src/common/container.rs

```diff
@@ -123,6 +125,18 @@ struct Inner {
     version: Version,
 }
 
+/// Find the resource container that best matches the given version requirement.
+pub fn find_resource(
```

Comment:
I'm fine with that `fn` but think it should not be placed in the `container` module. Maybe `State::match_container`?

Reply:
Moved in next commit.